app/src/main/res/layout/fragment_new_profile.xml

```diff
@@ -18,121 +22,113 @@
-                            android:layout_weight=".1" />
-
-                </LinearLayout>
-
-                <LinearLayout
-                        android:layout_width="match_parent"
-                        android:layout_height="wrap_content"
-                        android:layout_marginStart="35dp"
-                        android:layout_marginTop="50dp"
-                        android:layout_marginEnd="35dp"
-                        android:visibility="visible">
-
-                    <ca.antonious.materialdaypicker.MaterialDayPicker
-                            android:id="@+id/day_picker"
-                            android:layout_width="match_parent"
-                            android:layout_height="wrap_content" />
-                </LinearLayout>
-
-            </LinearLayout>
+                    android:layout_weight=".35"
+                    android:focusable="false"
+                    android:focusableInTouchMode="false"
+                    android:hint="@string/start_time"
+                    android:padding="10dp"
+                    android:textAlignment="center"
+                    android:textIsSelectable="false" />
+
+            <TextView
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_weight=".1" />
 
+            <EditText
+                    android:id="@+id/EndTime"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_weight=".35"
+                    android:focusable="false"
+                    android:focusableInTouchMode="false"
+                    android:hint="@string/end_time"
+                    android:padding="10dp"
+                    android:textAlignment="center"
+                    android:textIsSelectable="false" />
+
+            <TextView
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_weight=".1" />
+
+        </LinearLayout>
+
+        <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_below="@id/timeLayout"
+                android:id="@+id/dayLayout"
+                android:layout_marginStart="35dp"
+                android:layout_marginTop="50dp"
+                android:layout_marginEnd="35dp"
+                android:visibility="visible">
+
+            <ca.antonious.materialdaypicker.MaterialDayPicker
+                    android:id="@+id/day_picker"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content" />
         </LinearLayout>
 
-        <androidx.appcompat.widget.AppCompatButton
-                android:id="@+id/makeProfileFab"
-                android:layout_width="wrap_content"
+        <RelativeLayout
```

Comment:
why wrap the button with a relative layout?

Reply:
Otherwise the layout_gravity="bottom" was not working. 
I looked on the net, one of the solution was to set alignParentBottom="true", which was increasing the height of button and was showing unexpected behavior.
Other solution that I found was to wrap it in a relative layout and it worked fine. So I did.

Reply:
Okay

Reply:
> I guess you need to first stop tagging random people and asking them what should be done.
> Regarding the problem, as I mentioned earlier the problem is arising because of scrollview. You need to search online on how to make the scrollview cover the whole screen.
> I don't think I am complicating the issue by asking you to make a single layout file instead of two and I hope after seeing the end resultant code you will be able to realize that.

Sorry for the trouble. I will keep it in mind for future.